pkg/skaffold/debug/transform_go_test.go

```diff
@@ -208,7 +208,7 @@ func TestDlvTransformerApply(t *testing.T) {
 	}
 	for _, test := range tests {
 		testutil.Run(t, test.description, func(t *testutil.T) {
-			config, image, err := dlvTransformer{}.Apply(&test.containerSpec, test.configuration, identity)
+			config, image, err := dlvTransformer{}.Apply(&test.containerSpec, test.configuration, identity, []string{})
```

Comment:
Can just use `nil` to avoid allocating an object.
```suggestion
			config, image, err := dlvTransformer{}.Apply(&test.containerSpec, test.configuration, identity, nil)
```

Reply:
ah great to know